tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -4090,11 +4090,6 @@ lldbg_emit_common_block_mdnode(LL_DebugInfo *db, SPTR sptr)
   cmnblk_mdnode = lldbg_create_common_block_mdnode(
       db, scope_modnode, ll_get_md_null(), SYMNAME(sptr));
   db->cur_cmnblk_mdnode = cmnblk_mdnode;
-  cmnblk_gv_mdnode = lldbg_create_cmblk_gv_mdnode(db, cmnblk_mdnode, sptr);
-  slot = LL_MDREF_value(cmnblk_gv_mdnode) - 1;
-  node = db->module->mdnodes[slot];
```

Comment:
Looks like `slot` and `node` have become unused variables and hence the CI is failing.
```
/home/runner/work/flang/flang/tools/flang2/flang2exe/lldebug.cpp:4081:14: error: unused variable 'node' [-Werror,-Wunused-variable]
  LL_MDNode *node;
             ^
/home/runner/work/flang/flang/tools/flang2/flang2exe/lldebug.cpp:4082:12: error: unused variable 'slot' [-Werror,-Wunused-variable]
  unsigned slot;```

Reply:
Thanks Kiran. After the change, tests passed.